instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react';
+import "./CommentSection.css";
+
+class CommentSection extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      username:this.props.username,
+      text:this.props.text,
+      original:this.props.original
+    }
+  }
+
+  render(){
+    if(this.state.original === true){
+      return (<div className="Original">
+        <p className="Name" >{this.state.username}</p>
+        <p className="Comment" >{this.state.text}</p>
+      </div>
+      );
+    }
+    else{
+      return (<div className="CommentDiv">
```

Comment:
Make sure to introduce a line break after your opening Parens here - just makes it more readable.